### ISSUE TITLE
scripting: expose all qemu configs 

### ISSUE THREAD

**Issue by naveenrajm7** [utmapp/UTM#6933]
Currently, only a couple of the qemu configs ( `hypervisor` & `uefi` ) are exposed to the scripting bridge. 

Should be able to update the values of all qemu configs . Ex: `TPMDevice`

Example use case:
When one wants to create a VM for Windows 11, the default created VM through the scripting will not boot. 
Because, the Windows VM creation workflow in the UTM UI will turn on few configs which will make the Windows setup to boot.
Specifically, the newly created VM through the windows flow will turn on `RTCLocalTime`, `TPMDevice` and `UEFIBoot` , all of which are false by default in a newly created VM and currently there is no way to change them via scripting.

**Comment by osy** [comment]
Yes, agreed. This was more of just me being too lazy to add everything.
